steering-files/src/main/resources/org/hps/steering/recon/PhysicsRun2019MCRecon_KF_WithSpacing.lcsim

```diff
@@ -0,0 +1,175 @@
+        </driver>
+        <!-- SVT reconstruction drivers -->
+        <!-- Driver used to associate raw tracker hits to corresponding sensor. -->
+        <driver name="RawTrackerHitSensorSetup" type="org.lcsim.recon.tracking.digitization.sisim.config.RawTrackerHitSensorSetup">
+          <readoutCollections>SVTRawTrackerHits</readoutCollections>
+        </driver>
+        <!-- Fit the six raw samples and extract the amplitude and t0. --> 
+        <driver name="RawTrackerHitFitterDriver" type="org.hps.recon.tracking.RawTrackerHitFitterDriver">
+          <chiSqrThresh>.5</chiSqrThresh>
+          <doOldDT>1</doOldDT>
+	  <fitAlgorithm>Pileup</fitAlgorithm>
+          <fitTimeMinimizer>Migrad</fitTimeMinimizer>
+          <!--use this to correct for trigger time in MC instead of subtractTriggerTime-->   
+          <useTimestamps>true</useTimestamps>     
+          <!--offset to get times centered at 0 after timestamp correction-->                 
+	  <!-- set to 0 for unspaced; ~112 for spaced -->
+	  <tsCorrectionScale>120</tsCorrectionScale>
+          <!--correct for the SVT fit time offset...this should be on if <useTimingConditions> is turned on in readout-->        
+          <correctTimeOffset>true</correctTimeOffset>   
+          <!--per sensor shift...set false becasue it's not in readout sim-->       
+          <correctT0Shift>true</correctT0Shift>      
+          <!--use truth time for MC???  typically not used-->             
+          <useTruthTime>false</useTruthTime>    
+          <!--time of flight corrections-->              
+          <subtractTOF>true</subtractTOF>     
+          <!--set this false for MC, true for data-->              
+          <subtractTriggerTime>false</subtractTriggerTime>           
+          <!--per-strip timing correction from database...this should be on i f <useTimingConditions> is turned on in readout  -->  
+          <correctChanT0>false</correctChanT0>          
+          <isMC>true</isMC>
+          <debug>false</debug>
+        </driver>
+        <!-- 
+             Use the fitted raw tracker hits and create 1D clusters using a 
+             nearest neighbor algorithm. 
+        --> 
+        <driver name="TrackerHitDriver" type="org.hps.recon.tracking.DataTrackerHitDriver">
+             <neighborDeltaT>24.0</neighborDeltaT>
+            <neighborDeltaTSigma>3.0</neighborDeltaTSigma>
+            <saveMonsterEvents>false</saveMonsterEvents>
+            <thresholdMonsterEvents>400</thresholdMonsterEvents>
+            <clusterSeedThreshold>4.0</clusterSeedThreshold>
+            <doTimeError>1.0</doTimeError> 
+            <clusterNeighborThreshold>3.0</clusterNeighborThreshold>
+            <clusterThreshold>3.0</clusterThreshold> 
+            <doDeadFix>true</doDeadFix>
+            <doVSplit>true</doVSplit>
+            <debug>false</debug>
+        </driver>
+    
+        <driver name="ReconParticleDriver_Kalman" type="org.hps.recon.particle.HpsReconParticleDriver" > 
```

Comment:
Do we need to enable `makeCollerCols` in MC? I guess it may not make sense for the majority of MC, but thought to check.

Reply:
I added these just so we are consistent